tests/test_writers.py

```diff
@@ -122,20 +122,24 @@ def test_insert(self, writer):
         with writer:
             writer.write_table({
                 'name': 'foo_insert',
-                'headings': ['id', 'a', 'b', 'c'],
+                'headings': ['id', 'a', 'b', 'c', 's'],
                 'rows': [
-                    ['bizzle', 1, 2, 3],
-                    ['bazzle', 4, 5, 6],
+                    ['bizzle', 1, 2, 3, 'hi'],
+                    ['bazzle', 4, 5, 6, 'hello'],
+                    ['bozzle', 7, 8, 9, {'#text': 'test_text','@case_type': 'person','@relationship': 'child','id': 'nothing'}],
+                    ['buzzle', 1, 8, 7, {'@case_type': '', '@relationship': 'child', 'id': 'some_id'}],
```

Comment:
I actually think this should fail rather than return an empty result since it indicates that the source path is likely wrong